wiki/Glossary/en.md

```diff
@@ -432,7 +419,7 @@ The ignore list is a blacklist of words (or users) that the player does not want
 
 ### IRC
 
-IRC is an abbreviation for "Internet Relay Chat".
+IRC is an abbreviation for Internet Relay Chat.
 This is the protocol that osu! uses for its ingame chat.
 Because of this, you can use your favorite IRC client to connect to the ingame chat without needing to open the osu!client.
 Connect to `irc.ppy.sh` and you will auto-join the osu channel.
```

Comment:
Probably worth removing this section and substituting it for link to an article on the IRC connection process,